sdk/python/kubeflow/training/__init__.py

```diff
@@ -0,0 +1,48 @@
+# coding: utf-8
+
+# flake8: noqa
+
+"""
+    tensorflow
+
+    Python SDK for tensorflow  # noqa: E501
```

Comment:
minor: we may want to override annotation later. I remember it merge into first one instead of accepting a totally new name